flash-script.nix

```diff
@@ -39,6 +42,8 @@
   export NO_RECOVERY_IMG=1
   export NO_ESP_IMG=1
 
+  export ADDITIONAL_DTB_OVERLAY=${lib.concatStringsSep "," additionalDtbOverlays}
```

Comment:
This unfortunately means we can't pass ADDITIONAL_DTB_OVERLAY at runtime, can you have it append to the existing ADDITIONAL_DTB_OVERLASY if it exists?